src/main/scala/gemmini/Configs.scala

```diff
@@ -51,8 +51,7 @@ object GemminiConfigs {
     has_nonlinear_activations = true,
 
     // Reservation station entries
-    reservation_station_full_entries = 16,
-    reservation_station_partial_entries = 8,
+    reservation_station_entries_per_type = 16,
```

Comment:
Can we get separate lengths for each type (_execute_, _load_, and _store_)?

In general, we need a lot more _execute_ entries than we do for _load_ and _store_ entries.

Reply:
Done